### PR TITLE
Stack documentation URL

### PR DESCRIPTION
**Pull request by LDannijs**
#### Summary

Add a URL to The Things Industries documentation of integrations
@felixheins wanted this for the Partner Portal

#### Changes

- add `stack-documentation-url`
- Update broken URLs

#### Notes for Reviewers

@KrishnaIyer is this properly implemented?

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [x] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

**Comment by felixheins** [comment]
Thanks @LDannijs . Just to show an example: 

In the partner portal, we want to keep visitors on our pages and not lead them out. Not only for lead development reasons but also because partner links break and we can't control them. 

https://partners.thethingsindustries.com/integrations-details/datacake/r/reccQxsCgQ0jVm2zW

File: schema/lorawan_webhook_templates.go [hunk 1]
```diff
@@ -16,19 +16,20 @@ package schema
...
-	CreateDownlinkAPIKey bool                   `yaml:"create-downlink-api-key"`
-	Paths                WebhookTemplatePaths   `yaml:"paths,omitempty"`
-	FieldMask            []string               `yaml:"field-mask,omitempty"`
+	TemplateID            string                 `yaml:"template-id"`
+	Name                  string                 `yaml:"name"`
+	Description           string                 `yaml:"description"`
+	LogoURL               string                 `yaml:"logo-url"`
+	InfoURL               string                 `yaml:"info-url"`
+	DocumentationURL      string                 `yaml:"documentation-url"`
+	StackDocumentationURL string                 `yaml:"stack-documentation-url"`
```
Comment (KrishnaIyer) [inline schema/lorawan_webhook_templates.go:25]: The Things Stack; so maybe `TTSDocumentationURL`

**Review by KrishnaIyer — CHANGES_REQUESTED** [review]
Implementation looks ok to me. Some of these vendors (Ex: `iot-factory.yml`) don't have the new URL. Is that intended?

**Comment by LDannijs** [comment]
Yes some don't because they don't have any documentation.

**Review by KrishnaIyer — APPROVED** [review]

**Pull request merged**